packages/teams-js/test/public/app.spec.ts

```diff
@@ -496,9 +543,17 @@ describe('Testing app capability', () => {
           utils.respondToMessage(getContextMessage, contextBridge);
           const actualContext = await contextPromise;
 
+          const firstActionItem =
+            isM365ContentType(actualContext.actionInfo?.actionObjects[0]) && actualContext.actionInfo?.actionObjects[0];
+          const secondActionItem = actualContext.actionInfo?.actionObjects[1];
+
           expect(actualContext).toEqual(expectedContext);
           expect(actualContext.page.frameContext).toBe(context);
-          expect(actualContext.meeting.id).toBe('dummyMeetingId');
+          expect(actualContext.meeting?.id).toBe('dummyMeetingId');
+          expect(actualContext.actionInfo?.actionId).toBe('actionId');
+          expect(actualContext.actionInfo?.actionObjects.length).toBe(5);
+          expect(firstActionItem.secondaryId?.name).toEqual(SecondaryM365ContentIdName.DriveId);
+          expect(isM365ContentType(secondActionItem)).toBe(false);
```

Comment:
I wanted to test the typeguard and also add all of the different objects.  I didn't think it was necessary to actually iterate through the 5 objects to test for equality but maybe that is an oversight on my part.  I would be more than happy to add that if necessary